models/edr/alerts/error_models.sql

```diff
@@ -0,0 +1,23 @@
+{{
+  config(
+    materialized = 'view',
+    bind=False
+  )
+}}
+
+select model_execution_id as alert_id,
+       unique_id,
+       generated_at as detected_at,
+       database_name,
+       materialization,
+       path,
+       original_path,
+       schema_name,
+       message,
+       owner as owners,
+       tags,
+       alias,
+       status,
+       full_refresh
+from {{ ref('model_run_results') }}
+where {{ not elementary.get_config_var('disable_model_alerts') }} and lower(status) != 'success'
```

Comment:
Please verify that this will be treated as boolean on all platforms, you can maybe use the jinja filter as_bool and then cast it to bool using elementary.type_bool()

Reply:
I've tested this an it works as expected.
Basically if you'd provide falsely values (`false`, `0`, etc.) it treats them as such, and everything else would be truly.
We also this in [elementary_alerts_ignore_warn](https://github.com/elementary-data/dbt-data-reliability/blob/master/models/edr/alerts/alerts_dbt_tests.sql#L35).

Reply:
Did you test it on all platforms? and it's not true, in [here](https://github.com/elementary-data/dbt-data-reliability/blob/master/models/edr/alerts/alerts_dbt_tests.sql#L35) we don't add a bool condition with Jinja. We use the jinja to add a regular SQL condition